examples/api/lib/cupertino/picker/cupertino_picker.0.dart

```diff
@@ -37,7 +38,7 @@ class CupertinoPickerExample extends StatefulWidget {
 }
 
 class _CupertinoPickerExampleState extends State<CupertinoPickerExample> {
-  int _selectedFruit = 0;
+  int selectedFruit = 0;
```

Comment:
If you wouldn't make this a public accessor if the class were public, then don't make it a public accessor here. I know, this is an example, but it's what we'd do in production code.

Reply:
That makes sense. Thank you